app/models/manageiq/providers/ansible_tower/configuration_manager/playbook.rb

```diff
@@ -0,0 +1,4 @@
+class ManageIQ::Providers::AnsibleTower::ConfigurationManager::Playbook < ConfigurationScript
+  belongs_to  :configuration_project, class_name: "ManageIQ::Providers::AnsibleTower::ConfigurationManager::ConfigurationSource"
+  has_many    :configuration_script,  class_name: "ManageIQ::Providers::AnsibleTower::ConfigurationManager::ConfigurationScript"
```

Comment:
Ok, missed this for the renaming. 

So a `JobTemplate` has a `Playbook` and a `Playbook` can be used by many `JobTemplate`.  Hence `ConfigurationScript (1) : (n) ConfigurationScriptBase`.  (Are we all agree `ConfigurationScriptBase` as name for `Playbook`?)